src/main/java/com/blackducksoftware/integration/hub/service/model/ComponentVersionPolicyViolationCount.java

```diff
@@ -46,8 +46,8 @@ public ComponentVersionPolicyViolationCount(final NameValuePairView nameValuePai
             name = PolicySeverityType.valueOf(nameValuePair.name);
         }
 
-        if (StringUtils.isNumeric(nameValuePair.value)) {
-            value = Integer.valueOf(nameValuePair.value);
+        if (nameValuePair.value != null && NumberUtils.isCreatable(nameValuePair.value.toString())) {
```

Comment:
can you extract nameValuePair into a variable before the if statement then use use that might make it a little easier to read.

Object countJsonObject = nameValuePair.value;
if(countJsonObject != null && NumberUtils.isCreatable(countJsonObject.toString()) {
    value = NumberUtils.createNumber(countJsonObject.toString()).intValue();
}